platform/darwin/src/MGLHeatmapStyleLayer.h

```diff
@@ -40,6 +40,27 @@ MGL_EXPORT
 
 #pragma mark - Accessing the Paint Attributes
 
+/**
+ Defines the color of each point based on its density value in a heatmap. 
+ Should be an expression that uses `["heatmapDensity"]` as input.
```

Comment:
We’ll need to customize this documentation to use the `$heatmapDensity` syntax instead of the JSON syntax or remove this sentence entirely, since `$heatmapDensity` is mentioned below. Add an entry for `heatmap-color` to style-spec-overrides-v8.json.